packages/host/app/resources/stack-backgrounds.ts

```diff
@@ -46,6 +46,9 @@ export class StackBackgroundsResource extends Resource<Args> {
         }
         let bottomMostStackItem = stack[0];
         await bottomMostStackItem.ready;
+        if (bottomMostStackItem.cardError) {
+          return undefined;
```

Comment:
Is it impossible to know what realm an error doc is from?

Reply:
Hmm, I’ll look into it. Presumably some realm is returning the error response. Perhaps it’s a header?

Reply:
Good idea

Reply:
addressed in https://github.com/cardstack/boxel/pull/1834/commits/64e950c562812baca7c245484cb49a506f2f7fc0